### ISSUE TITLE
`conda skeleton pypi sendgrid` fails

### ISSUE THREAD

**Issue by ctaggart** [conda/conda-build#530]
I'm trying to build my first ever conda package and I'm getting this error when running `conda skeleton pypi sendgrid`:

```
Warning, the following versions were found for sendgrid
1.4.0
1.3.0
1.2.4
1.2.3
1.2.2
1.2.1
1.2.0
1.1.2
1.1.1
1.1.0
1.0.1
1.0.0
0.5.4
0.5.3
0.5.2
0.5.1
0.5.0
0.4.1
0.4.0
0.3.7
0.3.6
0.3.5
0.3.4
0.3.3
0.3.2
0.3.1
0.3.0
0.2.9
0.2.8
0.2.7
0.2.6
0.2.5
0.2.4
0.2.3
0.2.2
0.2.1
0.2.0
0.1.4
0.1.3
0.1.2
0.1.1
0.1.0
Using 1.4.0
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/s/sendgrid/sendgrid-1.4.0.tar.gz (7 KB) for sendgrid.
Downloading sendgrid
Using cached download
Unpacking sendgrid...
done
working in c:\users\camer\appdata\local\temp\tmperodx5conda_skeleton_sendgrid-1.4.0.tar.gz
Fetching package metadata: ......
conda : 69 unparsed bytes left at the end of stream
At line:1 char:1
+ conda skeleton pypi sendgrid 2>&1 > conda-skeleton-pypi-sendgrid.txt
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (69 unparsed byt...e end of stream:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
file 1/1:     core.py

 hunk no.1 doesn't match source file at line 165

  expected: 

  actual  :                 raise SystemExit, "error: " + str(msg)

source file is different - core.py

Solving package specifications: .
Applying patch: 'c:\\users\\camer\\appdata\\local\\temp\\tmperodx5conda_skeleton_sendgrid-1.4.0.tar.gz\\pypi-distutils.patch'
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda-build/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "C:\Miniconda\Scripts\conda-skeleton-script.py", line 4, in <module>
    sys.exit(main())
  File "C:\Miniconda\lib\site-packages\conda_build\main_skeleton.py", line 272, in main
    args_func(args, p)
  File "C:\Miniconda\lib\site-packages\conda_build\main_build.py", line 400, in args_func
    args.func(args, p)
  File "C:\Miniconda\lib\site-packages\conda_build\main_skeleton.py", line 286, in execute
    pypi.main(args, parser)
  File "C:\Miniconda\lib\site-packages\conda_build\pypi.py", line 341, in main
    get_package_metadata(args, package, d, data)
  File "C:\Miniconda\lib\site-packages\conda_build\pypi.py", line 478, in get_package_metadata
    python_version=args.python_version)
  File "C:\Miniconda\lib\site-packages\conda_build\pypi.py", line 739, in get_pkginfo
    run_setuppy(src_dir, tempdir, python_version)
  File "C:\Miniconda\lib\site-packages\conda_build\pypi.py", line 786, in run_setuppy
    apply_patch(join(stdlib_dir, 'distutils'), patch)
  File "C:\Miniconda\lib\site-packages\conda_build\source.py", line 238, in apply_patch
    check_call([patch, '-p0', '-i', path], cwd=src_dir)
  File "C:\Miniconda\lib\subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['C:\\Miniconda\\Scripts\\patch.bat', '-p0', '-i', 
'c:\\users\\camer\\appdata\\local\\temp\\tmperodx5conda_skeleton_sendgrid-1.4.0.tar.gz\\pypi-distutils.patch']' returned non-zero 
exit status -1
```

Here is my `conda info`:

```
Current conda install:

             platform : win-64
        conda version : 3.15.1
  conda-build version : 1.16.0
       python version : 2.7.9.final.0
     requests version : 2.7.0
     root environment : C:\Miniconda  (writable)
  default environment : C:\Miniconda
     envs directories : C:\Miniconda\envs
        package cache : C:\Miniconda\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
          config file : C:\Users\camer\.condarc
    is foreign system : False
```

Looking at the current open issues, this may be related to #476 & #478.

**Comment by remram44** [comment]
Duplicate of #119

**Comment by msarahan** [comment]
I believe these issues are fixed with the patch package, version 2.5.9, build 1. This bundles the GnuWin32 patch executable with a manifest that tells Windows to not require elevation when running patch.exe.

If you have patch.bat in your scripts folder, please remove it. All tracebacks using the proper program should show "patch.exe" not "patch.bat"

Closing this issue. If there are further issues with patch, please post new issues with tracebacks.
